curvefs/src/client/rpcclient/task_excutor.h

```diff
@@ -44,17 +44,22 @@ using ::curvefs::client::common::ExcutorOpt;
 using ::curvefs::client::common::MetaserverID;
 using ::curvefs::client::common::MetaServerOpType;
 using ::curvefs::common::PartitionInfo;
+using ::curvefs::metaserver::MetaStatusCode;
 
 namespace curvefs {
 namespace client {
 namespace rpcclient {
 
+class TaskExecutorDone;
+
+MetaStatusCode ConvertToMetaStatusCode(int retcode);
```

Comment:
it seems this function is only used in .cpp file, so no need to declare it in the header file.

Reply:
it was used by two different cpp files